docs/source/installation.rst

```diff
@@ -37,6 +37,10 @@ To install ``sktime`` with maximum dependencies, including soft dependencies, in
 
     pip install sktime[all_extras]
 
+.. warning::
+    Some of the dependencies included in ``all_extras`` do not work on mac ARM-based processors, such
+    as M1, M1Pro, M1Max or M1Ultra. This may cause an error during installation.
```

Comment:
do you know any workarounds that one could mention here, or in the troubleshooting section (if longer)?

Reply:
I could add the affected libraries and the fix for TensorFlow in the troubleshooting section if you think it is OK.

Reply:
yes, that would be great! Short note next to the install with a link to the troubleshooting section, where you can add more details and workarounds.

Reply:
I added a brief explanation in the workaround section. I hope it is not too much detail.